src/plist.c

```diff
@@ -2179,7 +2178,7 @@ Int             IsSSortPlist (
 	if (isHom)
 	  SET_FILT_LIST( list, FN_IS_HOMOG);
 	else
-	  SET_FILT_LIST( list, FN_IS_HOMOG);
+	  SET_FILT_LIST( list, FN_IS_NHOMOG);
```

Comment:
The commit message here just says "Fix typo in IsSSortPlist" which IMHO makes it sound completely harmless, but it really is an actual bug that's being fixed here, now? Perhaps "Fix bug in IsSSortPlist" or "Fix typo bug in IsSSortPlist" or so?

Reply:
You are right, this is a bug (although I believe caused by a typo, but many bugs are.